CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
-# New in 2020.6
+# Release Notes
+## New in 2020.7
+- Remove lots of old/unused stuff regarding remarks
```

Comment:
Where did you remove this? In previous commits?

Reply:
Yes, here:

https://github.com/isaqb-org/advanced-template/commit/aabe8a40c5c67f2c85927d70703483992047e2b9